src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {person.age
+      && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+      )
+    }
+    <p className="Person__partner">
+      {person.isMarried
+        ? `${person.partnerName} is my ${
+          person.sex === 'm' ? 'wife' : 'husband'}`
+        : 'I am not married'
+      }
```

Comment:
Can you avoid using nested ternary operators?
Could you think about the separation of this part of the code?